src/main/java/edu/hm/hafner/coverage/parser/CloverParser.java

```diff
@@ -108,7 +108,11 @@ private ModuleNode readCoverage(final String fileName, final XMLEventReader read
             if (event.isStartElement()) {
                 var startElement = event.asStartElement();
                 if (PROJECT.equals(startElement.getName())) {
-                    var projectName = getValueOf(startElement, NAME);
+                    var projectName = "-";
+		    var optionalName = getOptionalValueOf(startElement, NAME);
+		    if (optionalName != null) {
+			projectName = optionalName;
+		    }
```

Comment:
Let's simplify this a bit:
```suggestion
                    var projectName = getOptionalValueOf(startElement, NAME);
                    if (projectName == null) {
                        projectName = '-';
                    }
```

Reply:
Actually it is much easier: 

```suggestion
		    var projectName = getOptionalValueOf(startElement, NAME).orElse(EMPTY);
```